internal/controller/nodeagent.go

```
@@ -9,6 +9,10 @@ import (
 
 	"github.com/go-logr/logr"
 	configv1 "github.com/openshift/api/config/v1"
+	oadpv1alpha1 "github.com/openshift/oadp-operator/api/v1alpha1"
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gci)